<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1734</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Analysis</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1731</width>
     <height>891</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="single_tab">
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <attribute name="title">
     <string>Single Water Flow</string>
    </attribute>
    <widget class="QPushButton" name="select_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Static:</string>
     </property>
    </widget>
    <widget class="QLabel" name="dyn_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Dynamic:</string>
     </property>
    </widget>
    <widget class="QLabel" name="et_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Emptying Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="droop_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Droop:</string>
     </property>
    </widget>
    <widget class="QLabel" name="roi_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rate of Increase:</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_name_label">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>30</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Water Flow Data Name</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="surround_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Surroundings:</string>
     </property>
    </widget>
    <widget class="QLabel" name="x_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
    <widget class="QLabel" name="y_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>510</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
    <widget class="QLabel" name="static_change">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dyn_change">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="emp_change">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="dr_change">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="roi_change">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MplWidget" name="mplwid" native="true">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>1161</width>
       <height>671</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sel_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose a tank</string>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>610</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>630</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>650</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>670</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>690</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>710</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="x_label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>730</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>530</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>550</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>570</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>590</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>610</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>630</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>650</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>670</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>690</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>710</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="y_label_12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>730</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="sel_button_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose an injector</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sel_button_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose high pressure (don't click)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_data">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Data</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header location="global">mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
